instrumentation/apache-httpclient/apache-httpclient-4.3/library/src/main/java/io/opentelemetry/instrumentation/apachehttpclient/v4_3/ApacheHttpClientTracingBuilder.java

```diff
@@ -53,7 +53,7 @@ public ApacheHttpClientTracingBuilder addAttributeExtractor(
    * @param capturedHttpHeaders An instance of {@link CapturedHttpHeaders} containing the configured
    *     HTTP request and response names.
    */
-  public ApacheHttpClientTracingBuilder captureHttpHeaders(
+  public ApacheHttpClientTracingBuilder setCaptureHttpHeaders(
```

Comment:
Think this / and the type needs to be renamed to `setCapturedHttpHeaders`

Reply:
makes sense, I will apply everywhere